src/test/scala/annoy4s/AnnoySpec.scala

```diff
@@ -181,8 +181,8 @@ class AnnoySpec extends FlatSpec with Matchers {
 
     val index = Annoy.create[Int](tmpFile.pathAsString, numOfTrees = 2)
 
-    index.query(1, maxReturnSize = 10, searchK = 2).get.map(_._1) shouldBe List(1, 54, 55, 60, 76, 8, 32, 33)
-    index.query(1, maxReturnSize = 10, searchK = -1).get.map(_._1) shouldBe List(1, 69, 39, 87, 54, 29, 62, 55, 21, 35)
+    index.query(1, maxReturnSize = 10, searchK = 2).get.map(_._1) shouldBe List(1, 69, 39, 87, 54, 29, 62, 55, 21, 35)
```

Comment:
My C++ foo is terrible. If @pishen could help verify this, that would be great. 🙏🏽

Reply:
Fixed. This is buggy in 1.17.0 and I've now moved to 1.16.3